MH-13198, Properly Display Multiple Presenters

lkiesow opened the pull request:
If an event has multiple presenters, the main event table will display
them as one string, separated by comma. This can be a real usability
issue if the name format is in the form of “Doe, Jane”, where users will
end up with lines like “Doe, Jane, Doe, John, Mustermann, Max” which is
very confusing.

This patch tries to make the separate metadata entries visually distinct
by containing each entry in its own element, surrounded by a
lightweight border.

<img src=https://user-images.githubusercontent.com/1008395/47608529-59d13100-da2f-11e8-94aa-dc5561496bf2.png width=400px/>

lkiesow committed:
MH-13198, Properly Display Multiple Presenters

If an event has multiple presenters, the main event table will display
them as one string, separated by comma. This can be a real usability
issue if the name format is in the form of “Doe, Jane”, where users will
end up with lines like “Doe, Jane, Doe, John, Mustermann, Max” which is
very confusing.

This patch tries to make the separate metadata entries visually distinct
by containing each entry in it's own element, surrounded by a
lightweight border.

staubesv commented:
@lkiesow Any reason to not apply that style change to other multi-select fields (e.g. series.organizers, series.publishers, series.presenter)?

In theory, it would be desirable to also do this in the modals and wizards, but that might be out-of-scope for this PR.

staubesv reviewed (changes requested):
Should at least be done consistently on the tables - preferably also for the modals and wizards.

staubesv commented on modules/admin-ui/src/main/webapp/styles/extensions/components/_tables.scss:
```diff
@@ -25,6 +25,14 @@
         border-top-left-radius: 0;
         border-top-right-radius: 0;
+
+        .metadata-entry {
```
Maybe some more precise name for the style here? The table only contains "metadata entries", doesn't it?

lkiesow replied:
It's meant for a single metadata entry in a collection of metadata after all.
How would you prefer it to be named?

lkiesow replied:
@staubesv 
> How would you prefer it to be named?

lkiesow commented:
> Should at least be done consistently on the tables - preferably also for the modals and wizards.

That makes sense. I missed that the series table has the contributors and organizers fields. I will update this shortly.

lkiesow committed:
MH-13198, Properly Display Creators and Contributors

This patch will make creators and contributors in the series table being
displayed in the same manner presenters are in the events table.

lkiesow commented:
Updates series table as well.

I deliberately chose to not set this for roles in the user/groups tables. These aren't exactly displayed well as it and due to their special format, I don't think they would benefit from this. They are also a different type of metadata making this clash less with these adjustments.

lkiesow commented:
Fixed merge conflicts.
